### PR TITLE
Administration related topics

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/users-guide/administration-bundle/changing-settings.md [hunk 1]
```diff
@@ -516,14 +526,20 @@ Click **Properties library** in the **Settings** menu, to add custom properties
 
 With custom properties, you can extend the data model of {{< product-c8y-iot >}} built-in objects. You may create the following custom values:
 
-- Custom inventory properties are used to extend the inventory data model. They can be used in the "Asset table" and "Asset properties" widgets.
+- Custom inventory properties are used to extend the inventory data model. They can be used in the [Asset table widget](/users-guide/cockpit/#widget-asset-table) and [Asset property widget](/users-guide/cockpit/#asset-property).
 - Custom tenant properties are available during tenant creation. The custom properties can be edited under **Subtenants** in the **Custom properties** tab of each tenant. Additionally, these properties can be viewed and exported in the **Usage statistics**.
-- Custom alarm and event properties can be used as custom fields which can be added to your reports and will be available in the **Export** page in the Cockpit application.
+- Custom alarm and event properties can be used as custom fields which can be added to your reports and will be available in the **Exports** page in the Cockpit application.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/changing-settings.md:531]:
```suggestion
- Custom alarm and event properties can be used as custom fields which can be added to your reports and will be available in the **Export** page in the Cockpit application.
```

File: content/users-guide/administration-bundle/changing-settings.md [hunk 2]
```diff
@@ -41,6 +41,11 @@ Click **Authentication** in the **Settings** menu if you want to view or change
 To see the **Authentication** menu entry, you must have "Tenant management" ADMIN permission (`ROLE_TENANT_ADMIN` or `ROLE_TENANT_MANAGEMENT_ADMIN`).
 {{< /c8y-admon-info >}}
 
+{{< c8y-admon-related >}}
+- [Two-factor authentication](/users-guide/administration/#tfa) for details on the two-factor authentication strategies in {{< product-c8y-iot >}}.
+- [Authentication](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Authentication) in the {{< openapi >}} for details on managing authentication via REST.
```
Comment (manasesjesus) [inline content/users-guide/administration-bundle/changing-settings.md:46]:
```suggestion
- [Authentication](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#section/Authentication) in the {{< openapi >}} for details on managing authentication via REST.
```

File: content/users-guide/administration-bundle/data-retention.md [hunk 3]
```diff
@@ -13,11 +13,20 @@ helpcontent:
...
-You might however want to store measurements for 90 days for example, but delete alarms already after 10 days.
+**Retention rules** give you control on how long data is stored in your account. By default, all historical data is deleted after 60 days (configurable in the system settings by the platform administrator). You might however want to store measurements for 90 days for example, but delete alarms already after 10 days.
 
+{{< c8y-admon-info >}}
 Retention rules are usually run during the night. When you edit a retention rule, you will not see an immediate effect in the **Usage** section on the **Home** screen of the Administration application.
+{{< /c8y-admon-info >}}
+
+{{< c8y-admon-related >}}
+- [Enterprise tenant > Managing tenants > Tenant policies](/users-guide/enterprise-tenant/#tenant-policies) in the <i>User guide</i> for details on the creation of tenant policies and retention rules on tenant level.
+- [Retention](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Retention) in the {{< openapi >}} for details on managing retention rules via REST.
```
Comment (manasesjesus) [inline content/users-guide/administration-bundle/data-retention.md:24]:
```suggestion
- [Retention rules](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Retention-rules) in the {{< openapi >}} for details on managing retention rules via REST.
```

File: content/users-guide/administration-bundle/tfa.md [hunk 4]
```diff
@@ -4,12 +4,19 @@ title: Two-factor authentication
...
+
 There are two possible TFA strategies: SMS and TOTP. Only one of them can be active at a time.
 
 To check whether TFA is enabled for a certain user, go to the **Users** page and see the TFA status column right from the password strength column. A key icon indicates that TFA is enabled and by hovering over it you can see the strategy that is being used.
 
 ![TFA status](/images/users-guide/Administration/admin-user-tfa-enabled.png)
 
+{{< c8y-admon-related >}}
+- [Changing settings > Changing authentication settings](/users-guide/administration/#authentication) for information on how to configure TFA and other authentication settings.
+- [Authentication](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Authentication) in the {{< openapi >}} for details on managing authentication via REST.
```
Comment (manasesjesus) [inline content/users-guide/administration-bundle/tfa.md:16]:
```suggestion
- [Authentication](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#section/Authentication) in the {{< openapi >}} for details on managing authentication via REST.
```

**Review by manasesjesus — CHANGES_REQUESTED** [review]
fixing links to the OpenAPI

**Review by manasesjesus — APPROVED** [review]
links to the OpenAPI are looking good

**Review by MWindel — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Comment by BeateRixen** [comment]
@Airdroplet Hi Jarek, we would like to add some "related topics" at the top of each chapter to guide the users to other places of interest in the documentation. I have already added what I found useful but perhaps you have some more/better ideas? Could you please review and comment if you think we should add links? Thanks!

**Comment by BeateRixen** [comment]
> @Airdroplet Hi Jarek, we would like to add some "related topics" at the top of each chapter to guide the users to other places of interest in the documentation. I have already added what I found useful but perhaps you have some more/better ideas? Could you please review and comment if you think we should add links? Thanks!

https://cumulocity.atlassian.net/browse/MTM-44131 This is the ticket, which is assigned to me. Let me know if I should create a subtask for you or someone else from your team?

**Review by Airdroplet — APPROVED** [review]

File: content/users-guide/administration-bundle/business-rules.md [hunk 5]
```diff
@@ -12,6 +12,14 @@ helpcontent:
 
 Alarm mapping enables you to change the severity and text of alarms to adapt them to your business priorities. For example, a loss of the connection to a device is by default a MAJOR alarm but may be critical to you. To change this, add an alarm mapping to change alarms related to connection losses to CRITICAL.
 
+{{< c8y-admon-related >}}
+- [Device Management > Monitoring and controlling devices > Working with alarms](/users-guide/device-management/#alarm-monitoring) in the <i>User guide</i> for information on working with alarms in general.
+- [Alarms](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Alarms) in the {{< openapi >}} for details on managing alarms via REST.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/business-rules.md:17]:
```suggestion
- [Alarms](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Alarms) in the {{< openapi >}} for details on managing alarms via REST.
```

File: content/users-guide/administration-bundle/changing-settings.md [hunk 6]
```diff
@@ -41,6 +41,11 @@ Click **Authentication** in the **Settings** menu if you want to view or change
 To see the **Authentication** menu entry, you must have "Tenant management" ADMIN permission (`ROLE_TENANT_ADMIN` or `ROLE_TENANT_MANAGEMENT_ADMIN`).
 {{< /c8y-admon-info >}}
 
+{{< c8y-admon-related >}}
+- [Two-factor authentication](/users-guide/administration/#tfa) for details on the two-factor authentication strategies in {{< product-c8y-iot >}}.
+- [Authentication](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#section/Authentication) in the {{< openapi >}} for details on managing authentication via REST.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/changing-settings.md:46]:
```suggestion
- [Authentication](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#section/Authentication) in the {{< openapi >}} for details on managing authentication via REST.
```

File: content/users-guide/administration-bundle/data-retention.md [hunk 7]
```diff
@@ -13,11 +13,20 @@ helpcontent:
...
-You might however want to store measurements for 90 days for example, but delete alarms already after 10 days.
+**Retention rules** give you control on how long data is stored in your account. By default, all historical data is deleted after 60 days (configurable in the system settings by the platform administrator). You might however want to store measurements for 90 days for example, but delete alarms already after 10 days.
 
+{{< c8y-admon-info >}}
 Retention rules are usually run during the night. When you edit a retention rule, you will not see an immediate effect in the **Usage** section on the **Home** screen of the Administration application.
+{{< /c8y-admon-info >}}
+
+{{< c8y-admon-related >}}
+- [Enterprise tenant > Managing tenants > Tenant policies](/users-guide/enterprise-tenant/#tenant-policies) in the <i>User guide</i> for details on the creation of tenant policies and retention rules on tenant level.
+- [Retention rules](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Retention-rules) in the {{< openapi >}} for details on managing retention rules via REST.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/data-retention.md:24]:
```suggestion
- [Retention rules](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Retention-rules) in the {{< openapi >}} for details on managing retention rules via REST.
```

File: content/users-guide/administration-bundle/managing-applications.md [hunk 8]
```diff
@@ -43,8 +58,21 @@ The {{< product-c8y-iot >}} platform distinguishes between applications and micr
...
+- [Managing permissions](#managing-permissions) for details on assigning roles and permissions for the usage of {{< product-c8y-iot >}} applications.
+- [Changing application settings](/users-guide/administration/#default-app) for information on changing the application settings for your account.
+- [Enterprise tenant > Managing tenants > Applications](/users-guide/enterprise-tenant/#applications) for information on application subscriptions on tenant level.
+- [Cockpit > Widgets collection > Application](/users-guide/cockpit/#applications) for information on the "Application" widget.
+- [Developing applications](/concepts/applications) in the *Concepts guide* for an overview on the basic concepts of applications in {{< product-c8y-iot >}}.
+- The [Web SDK guide](/web/overview) for information on how to develop web applications on top of {{< product-c8y-iot >}} and how to [customize](/web/application-configuration) or [extend](/web/tutorials/#extend-an-existing-application) existing applications using the Web SDK.
+- [Applications](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Applications) in the {{< openapi >}} for managing applications via REST.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-applications.md:73]:
```suggestion
- [Applications](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Applications) in the {{< openapi >}} for managing applications via REST.
```

File: content/users-guide/administration-bundle/managing-microservices.md [hunk 9]
```diff
@@ -20,6 +20,17 @@ Click **Microservices** in the **Ecosystem** menu in the navigator to display a
...
+- [Changing application settings](/users-guide/administration/#default-app) for information on changing the application settings for your account.
+- [Enterprise tenant > Managing tenants > Applications](/users-guide/enterprise-tenant/#applications) for information on application subscriptions on tenant level.
+- [Enterprise tenant > Usage statistics and billing](/users-guide/enterprise-tenant/#usage-and-billing) for information on the microservice usage feature.
+- [Developing applications > Microservices](/concepts/applications/#microservices) in the *Concepts guide* for an overview on the basic concepts of microservices in {{< product-c8y-iot >}}.
+- The [Microservice SDK guide](/microservice-sdk/introduction) for general aspects of using microservices on top of {{< product-c8y-iot >}} and information on developing and deploying microservices using our SDKs or the REST interface.
+- [Applications](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Applications) in the {{< openapi >}} for managing applications via REST.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-microservices.md:31]:
```suggestion
- [Applications](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Applications) in the {{< openapi >}} for managing applications via REST.
```

File: content/users-guide/administration-bundle/managing-permissions.md [hunk 10]
```diff
@@ -23,6 +23,13 @@ The following types of roles can be associated with users:
 
 Moreover, application access can be granted to enable a user to use an application.
 
+{{< c8y-admon-related >}}
+- [Managing users](#managing-users) for information on managing users in general.
+- [Managing applications](#managing-applications) for more information on managing applications.
+- [Roles](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Roles) and [Inventory Roles](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Inventory-Roles) in the {{< openapi >}} for managing permissions via REST.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-permissions.md:29]:
```suggestion
- [Roles](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Roles) and [Inventory Roles](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Inventory-Roles) in the {{< openapi >}} for managing permissions via REST.
```

File: content/users-guide/administration-bundle/managing-users.md [hunk 11]
```diff
@@ -36,6 +51,12 @@ Moreover, password reset in {{< product-c8y-iot >}} is disabled for users create
 Users which are using single sign-on cannot change the password of users which are managed by the platform.
 {{< /c8y-admon-info >}}
 
+{{< c8y-admon-related >}}
+- [Managing permissions](#managing-permissions) for details on assigning roles and permissions to users.
+- [Tenant hierachy](/concepts/tenant-hierarchy) in the *Concepts guide* for information on the concept of multi-tenancy as opposed to user access control.
+- [Users](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Users) in the {{< openapi >}} for managing users via REST.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-users.md:57]:
```suggestion
- [Users](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Users) in the {{< openapi >}} for managing users via REST.
```

File: content/users-guide/administration-bundle/tfa.md [hunk 12]
```diff
@@ -10,6 +10,12 @@ To check whether TFA is enabled for a certain user, go to the **Users** page and
 
 ![TFA status](/images/users-guide/Administration/admin-tfa-sms.png)
 
+{{< c8y-admon-related >}}
+- [Changing settings > Changing authentication settings](/users-guide/administration/#authentication) for information on how to configure TFA and other authentication settings.
+- [Authentication](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#section/Authentication) in the {{< openapi >}} for details on managing authentication via REST.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/tfa.md:15]:
```suggestion
- [Authentication](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#section/Authentication) in the {{< openapi >}} for details on managing authentication via REST.
```

**Review by rssag — APPROVED** [review]

**Review by MWindel — APPROVED** [review]

**Pull request merged**